### DISCUSSION TITLE
Files Thumbnail icons

### DISCUSSION THREAD

**Discussion by KuroProjects** [spencerwooo/onedrive-vercel-index#536] [Q&A] [answered]
I am getting confused in how to get icons like these in my index too
![image](https://user-images.githubusercontent.com/92580272/158061200-0676d271-33a6-40aa-a6db-382866ce8435.png)

**Comment by zkeq** [comment, marked as answer]
oh, just add `emoji` to the front of your folder or file name.

Just like this . 

```
🍏 Genshin emotes
  ^
(Note that there is a space..)
```

Emoji can be found here: http://emojipedia.org
